.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,11 @@
+blank_issues_enabled: false
```

Comment:
I think we should keep a section to report bugs in the templates. Although I don't expect this to be used frequently, it feels a bit weird to click on the "Add issue" button and not have the option to actually add an issue.

Reply:
You're right, I wasn't aware of that :D

I force-pushed and updated my screenshot above :+1: